contrib/security/PSS/dynamic/baseline/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+resources:
+- ../../../../../apps/profiles/upstream/base
+
+configMapGenerator:
+- name: namespace-labels-data
+  namespace: kubeflow
+  behavior: replace
+  files:
+  - namespace-labels.yaml
+  # options:
```

Comment:
# options:
  #   disableNameSuffixHash: true

this must be set on the profile controller base level